alembic/env.py

```diff
@@ -13,6 +15,7 @@
 
 # add your model's MetaData object here
 # for 'autogenerate' support
+sys.path.append(os.getcwd())
```

Comment:
I would somewhat prefer this, because I don't know how `alembic` handles cwd when it's called from other directories.

```python
sys.path.insert(0, path.realpath(path.join(path.dirname(__file__), '..')))
```

Reply:
Yeah, good catch. Thanks.